colorization/src/pipeline.py

```diff
@@ -76,18 +77,23 @@ def colorize_image(image_path_input, image_path_output):
         return FAILED
 
     #  inference & colorize
-    inference_output_path = 'inference_output.npy'
+    tmpdir = tempfile.TemporaryDirectory(suffix="_npy", prefix="tp_inference_",
+                                         dir="/tmp")
+    inference_output_path = tmpdir.name + '/inference_output.npy'
```

Comment:
looking good so far.
would be nice if you could use os.path.join here like you did in the tests.py